webmin/os-eol.map

```diff
@@ -0,0 +1,1810 @@
+=pod
```

Comment:
Is this just the file that the API provides?

Reply:
The API doesn't provide this. I made it as a reference to remember what the data structures are like and what Webmin thinks about OSes.

Reply:
So is this file used by any Webmin code? I'm confused as to why we need to check it in ..

Reply:
No, this is just a .map file. It won't go to the package itself. I checked it in for the reference!

Reply:
I think we should remove it, as it's kind of confusing.

Reply:
Alright, it's done. I just wanted to make it easier for future support.